store.html

```diff
@@ -0,0 +1,157 @@
+                            <input class='input' type="text" id="address-two">
+
+                            <div class='form-flex'>
+
+                                <div class='width-575'>
+                                    <label class='label' for='city'>City *</label>
+                                    <select class='select input' required id='city' form='order-form'>
+                                        <option value="" disabled selected>Select your city...</option>
+                                        <option value='london'>London</option>
+                                        <option value='budapest'>Budapest</option>
+                                        <option value='new-york'>New York</option>
+                                    </select>
+                                </div>
+                                
+                                <div class='width-375'>
+                                    <label class='label' for='postcode'>Postcode *</label>
+                                    <input type="text" class="input" id="postcode" required minlength=4 maxlength=8>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </fieldset>
+
+                    <fieldset class='hide-this margin-top-15'>
+                        <legend class='margin-bottom-1 show-this'>Select a colour</legend>
+
+                        <div class='show-this'>
+
+                            <input class='store-radio' type="radio" required value='karma-orange' name='colour' id='karma-orange'>
+                            <label class='margin-right' for='karma-orange'>Karma Orange</label>
+
+                            <input class='store-radio' type="radio" required value='space-grey' name='colour' id='space-grey'>
+                            <label class='' for='space-grey'>Space Grey</label>
+
+                        </div>
+                    </fieldset>
+                    
+                    <fieldset class='hide-this margin-top-3'>
+                        <legend class='hide-this'>Consent Box</legend>
+
+                        <div class='show-this'>
+
+                            <input class='checkbox' type='checkbox' id='consent-checkbox' value='consented' name='consent' required=true>
+                            <label for='consent-checkbox' class='consent-label'>By placing your order you agree to Karma's <a class='consent-link' href='#'>Terms and Conditions</a>. *</label>
+
+                        </div>
+                    </fieldset>
+
+                    <input class='margin-top-15 hero-btn hero-text hero-btn-a submit' type='submit' value='Place my order'>
+
+                </form>
+
+            </section>
+
+            <!-- Image -->
+
+            <img class="store-img right-flex-item" src="img/store-image_by-andrew-neel-unsplash.jpg" alt="woman working on her laptop while enjoying a hot beverage">
```

Comment:
It is not advisable to directly embed large images in your HTML code, as they slow down page load. Also, in this case, your image stretches and becomes distorted when resizing the browser page.

It would be better to instead treat this image as a background element of div or element, that you can set from CSS with background-size: cover.

See more info here: https://developer.mozilla.org/en-US/docs/Web/CSS/background-size